### PR TITLE
DOC: Add JOSS metadata

### PR DESCRIPTION
**Pull request by thewtex**
Add Journal of Open Source Software metadata per

  http://joss.theoj.org/about#paper_structure

@mlp6 @aylward @jcfr please take a look

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ - name: Duke University
+   index: 2
+date: 15 December 2016
+bibliography: paper.bib
+---
+
+# Summary
+
+A 3D Slicer [@Fedorov2012] extension for ultrasound image formation,
+processing, and analysis. Interfaces built off the ITKUltrasound library
```
Comment (jcfr) [inline paper.md]: `Interfaces built off the ` -> `Interfaces are built off the `

File: paper.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+[@McCormick2014]. The modules available in this extension are useful for both
+2D and 3D image generation for traditional B-mode imaging [@McCormick2010],
+but also next-generation ultrasound image modalities like ultrasound
+spectroscopy [@Aylward2016], acoustic radiation force imaging (ARFI)
+[@Palmeri2016], acoustic radiation force shear wave imaging (ARFI-SWEI), and
```
Comment (jcfr) [inline paper.md]: Should a reference for `ARFI-SWEI` be added ?  May be something like this:

```
+spectroscopy [@Aylward2016], acoustic radiation force imaging (ARFI)
+[@Palmeri2016], acoustic radiation force shear wave imaging (ARFI-SWEI) [@Palmeri2016], and
+others.
```

Also wonder if this publication should be cited: http://dukespace.lib.duke.edu/dspace/handle/10161/9062

**Comment by jcfr** [comment]
Beside of few nitpicks, LGTM :+1:

File: paper.md (hunk 1 above)
Comment (thewtex) [reply paper.md]: Done

File: paper.md (hunk 2 above)
Comment (thewtex) [reply paper.md]: Good idea.  Added suggested reference. We could use another too if @mlp6 prefers.

**Review by jcfr — APPROVED** [review]

**Comment by thewtex** [comment]
Submission: http://joss.theoj.org/papers/746880f9243feff57502fae59abce923

**Pull request merged**